tests/install_upgrade_operators/deployment/conftest.py

```
@@ -27,6 +30,37 @@ def cnv_deployments_excluding_hpp_pool(admin_client, hco_namespace):
+
+@pytest.fixture(scope="session")
+def jira_92888_open():
+    return is_jira_open(jira_id="CNV-92888")
+
+
+@pytest.fixture(scope="session")
+def jira_92889_open():
+    return is_jira_open(jira_id="CNV-92889")
+
+
+@pytest.fixture()
```

Ruff (0.15.21)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)